go.mod

```diff
@@ -3,54 +3,62 @@ module github.com/cosmos/interchain-security
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.11
+	github.com/confio/ics23/go v0.9.0
```

Comment:
Please stop refactoring in the same PRs where changes are introduces as it makes it difficult to review.

Reply:
Clearly `github.com/confio/ics23/go v0.9.0` was not added by this PR. It was just moved around the file which makes the review difficult. The contributions in a single PR should be minimal to improve the review process and avoid unwanted bugs.

Reply:
@mpoke - SDK v0.45.15 does add that.

It is not a refactor. 

It is a part of changing the sdk version.

Reply:
go mod tidy organizes files in go

Reply:
Oh that's right I forgot I definitely got rid of the weird artificial group in the middle of the direct imports because if it's there there should be a comment explaining it but it's just sort of there so I got rid of it